### DISCUSSION TITLE
Lisensi

### DISCUSSION THREAD

**Discussion by hendroprwk08** [nafiesl/silsilah#102] [General]
Hai kak, untuk membeli aplikasi ini harganya berapa ya? nanti lisensi dialihkan menjadi atas nama saya

**Comment by nafiesl** [comment]
Halo mas @hendroprwk08 sudah terjawab ya mas, dari diskusi kita di email. Sedikit saya rangkum:

1. Saya belum ada niat menjual project ini.
2. Untuk kebutuhan project yang mas rencanakan, silakan clone/fork repository project ini, kemudian mas bisa perbaiki dan modifikasi sesuai kebutuhan project mas.
3. Project ini boleh di download, dipelajari, dimodifikasi, didistribusi, dijual, atau dibisniskan dengan atau tanpa minta ini kepada saya selaku pembuatnya, selama file [LICENSE](https://github.com/nafiesl/silsilah/blob/master/LICENSE) **tidak diubah atau dihapus** dari source codenya (sesuai ketentuan/isi lisensi MIT).

Tambahan:

1. Penjelasan tentang lisensi MIT: https://www.youtube.com/watch?v=P0q1X7WBTkE

Terima kasih.
